analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Constants.cs

```diff
@@ -0,0 +1,84 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.SymbolicExecution.Constraints;
+using SonarAnalyzer.UnitTest.TestFramework.SymbolicExecution;
+
+namespace SonarAnalyzer.UnitTest.SymbolicExecution.Roslyn
+{
+    public partial class RoslynSymbolicExecutionTest
+    {
+        [TestMethod]
+        [DataRow("true")]
+        [DataRow("false")]
+        public void PreProcess_TrueLiteral_BooleanConstraint(string value)
+        {
+            var expectedConstraint = GetBooleanConstraint(value);
```

Comment:
I'd do it like this. The `Get` doesn't explain where does it take the constraint from.
```suggestion
            var expected = ExpectedBooleanConstraint(value);
```

Reply:
It "computes" the constraints based on the parameter. I prefer to have it this way.

Reply:
Then it would  be `ComputeBooleanConstraint`. But I still prefer the `ExpectedBooleanConstraint`. The `Get` confuses me as it isn't obvious from where it gets the values

Reply:
The way I see it the methods/functions are actions that do something and a proper method name should have a verb that describes the action.  `ExpectedBooleanConstraint` is lacking the verb, and the prefix "Expected" does not tell anything about what the method does.